mods/qtcore/nodeboxes.lua

```diff
@@ -74,4 +74,13 @@ qtcore.nb_dustpile = function()
 			{ -2/16, -8/16, -6/16, 2/16, -7/16, 6/16, },
 		}
 	}
+end
+
+qtcore.nb_dustpile_selection = function()
+	return{

```

Comment:
There should be a space between "return" and the bracket:
`return {`

Reply:
was an issue with all in file, all fixed